### ISSUE TITLE
Client crash when i try to interact with a trader

### ISSUE THREAD

**Issue by TarikYuzkiv** [anegostudios/VintageStory-Issues#1905]
**Game Version:** 1.16.5
**Platform:** Windows  
**Modded:** No
**SP/MP:** Multiplayer

The client is crashes when I try to buy something from a trader, it probably happens because the trader has 0 gears.
https://www.youtube.com/watch?v=2MdX61rTv64

**Comment by copygirl** [comment]
Could you post the crash?

**Comment by Konfuz** [comment]
I think I had the same thing happen to me just now. Sever Exception and I was on a multiplayer-server (version 1.17.9) so I don't have the server-log (asked an Admin for it). The Trader had 0 gears which I just noticed when  I clicked sell and the time for new offers was "less than a day" - after relog the trader had new offers (and about 40 gears) so I think it may have been a sync-issue with the trader having switched offers mid-trade session. 

I also had a chest open at the time I got disconnected - a while after reconnecting I had some issues with "disappearing items" and later found them in the chest next to the trader - which was at least a few hundred blocks away from where they "disappeared" from my inventory.
